engines/gob/util.cpp

```diff
@@ -671,8 +671,8 @@ Common::String Util::readString(Common::SeekableReadStream &stream, int n) {
 	return str;
 }
 
-/* NOT IMPLEMENTED */
 void Util::checkJoystick() {
+	warning("Util::checkJoystick: - NOT IMPLEMENTED");
 	_vm->_global->_useJoystick = 0;
 }
```

Comment:
I believe this is intentionally left unimplemented, but @sdelamarre can provide more insights

Reply:
I do not know what this joystick thing is about, but it is not called anywhere currently in the Gob engine currently.